src/main/java/me/zeroeightsix/kami/module/modules/player/PortalGodMode.java

```diff
@@ -0,0 +1,25 @@
+package me.zeroeightsix.kami.module.modules.player;
+
+import me.zero.alpine.listener.EventHandler;
+import me.zero.alpine.listener.Listener;
+import me.zeroeightsix.kami.event.events.PacketEvent;
+import me.zeroeightsix.kami.module.Module;
+import net.minecraft.network.play.client.CPacketPlayer;
+
+import java.util.LinkedList;
+import java.util.Queue;
+
+/**
+ * Created by GlowskiBroski on 10/14/2018.
+ */
+@Module.Info(name = "PortalGodMode", category = Module.Category.PLAYER)
+public class PortalGodMode extends Module {
+
+
+    @EventHandler
+    public Listener<PacketEvent.Send> listener = new Listener<>(event -> {
+       if (isEnabled() && event.getPacket() instanceof CPacketConfirmTeleport) {
```

Comment:
You're using `CPacketConfirmTeleport`, but not importing it - this is causing the build to fail.
Also, does ignoring this packet have any other sideeffects besides the godmode? I'd love to minimize those, if so.